### PR TITLE
LPS-x Remove unnecessary setting value and resetting.

### PR DESCRIPTION
**Pull request by tinatian**
Hi Marcos,

I am working on performance tuning, and found that we are doing unnecessary setting and resetting in the portal_normal.ftl.

I am not sure how these theme work and do not know how to test it, so I would like to check with you if my change would break any logic.

Thanks.
Tina.

### CODE REVIEW

**Comment by tinatian** [comment]
cc @Preston-Crary

File: modules/apps/foundation/frontend-theme/frontend-theme-user-dashboard/src/templates/portal_normal.ftl [hunk 1]
```diff
@@ -52,15 +52,11 @@
 					</#if>
 				</div>
 
-				<#assign VOID = freeMarkerPortletPreferences.setValue("portletSetupPortletDecoratorId", "barebone") />
```
Comment (marcoscv-work) [inline modules/apps/foundation/frontend-theme/frontend-theme-user-dashboard/src/templates/portal_normal.ftl:55]: Hello!

If we remove these lines the design will be different than originally was designed, so.. If this is a real and big performance problem, please let me know and we will try to get the design in other way.

Thanks a lot!

Comment (Preston-Crary) [reply modules/apps/foundation/frontend-theme/frontend-theme-user-dashboard/src/templates/portal_normal.ftl:55]: @marcoscv-work,

we're just trying to understand the current design.  Are these lines needed by the taglib?

Comment (marcoscv-work) [reply modules/apps/foundation/frontend-theme/frontend-theme-user-dashboard/src/templates/portal_normal.ftl:55]: Hey guys!

After a deeper look I think we can go ahead with this.
Right now user_personal_bar has not portlet container border, so we cannot configure its portlet decorator.

Thanks a lot!

**Comment by marcoscv-work** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by marcoscv-work** [comment]
If you guys want can send me the PR with the final LPS and I will resend to Brian

**Comment by Preston-Crary** [comment]
Sounds good, thanks!

**Comment by marcoscv-work** [comment]
Pull request submitted to https://github.com/eduardoallegrini/liferay-portal/pull/63. [See changes here.](https://github.com/marcoscv-work/liferay-portal/compare/17c7f48011a5~1...17c7f48011a5) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**